escrow/contracts/escrow.sol

```diff
@@ -0,0 +1,91 @@
+// SPDX-License-Identifier: Unlicensed
+pragma solidity ^0.8.9;
+
+import "@acala-network/contracts/token/Token.sol";
+
+contract Escrow {
+    address payable public requestor;
```

Comment:
I think that `requestor` could be renamed into `initiator` to be in line with the current tutorials.

Also, is there a reason for this to be `payable`?